pkg/gardenlet/controller/federatedseed/network-policy/networkpolicies_controller.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package network_policy
+
+import (
+	networkingv1 "k8s.io/api/networking/v1"
+)
+
+func (c *Controller) networkPolicyUpdate(_, newObj interface{}) {
+	policy := newObj.(*networkingv1.NetworkPolicy)
+	c.namespaceQueue.Add(policy.Namespace)
```

Comment:
Usually, keys created via the `cache.MetaNamespaceKeyFunc` function are added to queues. Why not here? (same for `networkPolicyDelete` function)

Reply:
We do not need a key for the NetworkPolicy, but for the namespace. That's why we can just add the policy.namespace.